docs/intro.md

```diff
@@ -1,109 +1,36 @@
 ---
 sidebar_position: 1
 sidebar_label: Introduction
-description: Intro to Rollkit, a modular development kit for building rollup chains.
+description: Intro to Rollkit, a modular rollup framework.
 ---
 
 # Introduction to Rollkit
 
-Rollkit is a modular development kit for anyone to easily deploy their own
-blockchain with minimal overhead. It solves the problem of bootstrapping a
-secure and decentralized validator set while removing the burden of maintaining
-a new consensus network.
+Rollkit is a rollup framework that gives developers the freedom to
```

Comment:
"Rollkit is a modular rollup framework that gives developers the freedom to deploy throughout the modular stack, opening new possibilities of rapid experimentation and innovation"

Reply:
we don't need "modular" twice in one sentence